packages/framework/src/Form/Constraints/MoneyRange.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form\Constraints;
+
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class MoneyRange extends Constraint
+{
+    /**
+     * @var string
+     */
+    public $minMessage = 'The amount of money should be {{ limit }} or more.';
+
+    /**
+     * @var string
+     */
+    public $maxMessage = 'The amount of money should be {{ limit }} or less.';
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Money\Money|null
+     */
+    public $min;
```

Comment:
I definitelly exptect `protected` here

Reply:
Constraints must have their properties public. For example, take a look at `\Symfony\Component\Validator\Constraints\Range`.

Reply:
My eyes bleeds https://github.com/symfony/symfony/blob/v4.2.3/src/Symfony/Component/Validator/Constraint.php#L126